divi/src/masternode.cpp

```diff
@@ -643,10 +642,10 @@ bool CMasternodeBroadcastFactory::checkMasternodeCollateral(
     const std::string& txHash,
     const std::string& outputIndex,
     const std::string& service,
-    CMasternode::Tier& nMasternodeTier,
+    MasternodeTier nMasternodeTier,
```

Comment:
This should stay by reference. I should have renamed this function to be `assignTierFromMasternodeCollateral` as it not only checks the value but assigns the corresponding tier.

Reply:
Good catch, fixed.